<launch>
  <arg name="robot" default="odroid0" />
  <arg name="camera" default="lifecam_$(arg robot)" />
  <arg name="device" default="/dev/video0" />

  <node ns="$(arg robot)" name="$(arg camera)" pkg="usb_cam" type="usb_cam_node">
    <param name="video_device" value="$(arg device)" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="mjpeg" />
    <param name="camera_frame_id" value="$(arg camera)" />
    <param name="camera_name" value="$(arg camera)" />
    <param name="io_method" value="mmap" />
    <param name="camera_info_url" value="package://multi_robot_fusion/calib/$(arg camera).yml" />
  </node>
</launch>
